src/main/java/org/apache/commons/lang3/StringEscapeUtils.java

```diff
@@ -413,6 +447,47 @@ public int translate(final CharSequence input, final int index, final Writer out
         }
     }
 
+    public static final CharSequenceTranslator UNESCAPE_XSI = new BackslashUnescaper();
+
+    /**
+     * Translator object for unescaping backslash escaped entries.
+     *
+     * @since 3.6
+     */
+    static class BackslashUnescaper extends CharSequenceTranslator {
```

Comment:
No sure why the author of the patch choose this naming. I'm fine with changing it.